### PR TITLE
Small packaging changes

### PR DESCRIPTION
**Pull request by D4N**
- don't use rpm's internal macros (prefixed with two _)
- use %buildroot instead of $RPM_BUILD_ROOT
- don't redefine %python_sitelib
- fix changelog

### CODE REVIEW

File: SPECS/mote.spec [hunk 1]
```diff
@@ -1,15 +1,13 @@
-%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from
-%distutils.sysconfig import get_python_lib; print (get_python_lib())")}
 %global srcname mote
 
 Name:           %{srcname}
 Version:        0.6.3
-Release:        4%{?dist}
+Release:        1%{?dist}
 Summary:        A MeetBot log wrangler, providing a user-friendly interface for Fedora's logs
```
Comment (nasirhm) [inline SPECS/mote.spec:6]: Note: It'll be: 1.fc32 (in case of Fedora 32).

**Review by nasirhm — APPROVED** [review]
Thank You Dan for reviewing the SPEC file and sharing the fix. :)

**Comment by nasirhm** [comment]
> * don't redefine %python_sitelib

Can you please point the line, where I was redefining it ?

**Comment by nasirhm** [comment]
Ah,, I've got it, I was trying to redefine the macro with python, It could be the __python macro or something related to that line.

File: SPECS/mote.spec [hunk 2]
```diff
@@ -1,15 +1,13 @@
-%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from
```
Comment (D4N) [inline SPECS/mote.spec:1]: You were redefining it here, albeit now that I look at it, it should be only added when not yet defined.

Comment (nasirhm) [reply SPECS/mote.spec:1]: Got it, Thank you very much Dan for the PR. :D

**Pull request merged**